### PR TITLE
add dev dashboard

### PR DESCRIPTION
**Pull request by andreipope**
add dev dashboard

### CODE REVIEW

File: docs/en/intro-to-loom.md [hunk 1]
```diff
@@ -185,3 +185,126 @@ yarn serve
...
+![Map Accounts](/developers/img/deploy-to-loom-dashboard.png)
+
+
+### Deposit Loom to Plasmachain
+
+From the left pane, click on the "Deploy to Plasmachain". You will be redirected to a page similar to the one below:
+
+![New Account Options](/developers/img/deploy-to-loom-deposit-loom.png)
+
+Notice that, in order to deploy to Plasmachain, you need to stake 10 LOOM. Click on the "Deposit more Loom to Plasmachain" and follow the instructions.
```
Comment (enlight) [inline docs/en/intro-to-loom.md]: Lets not mention any specific amount, that's subject to change.

File: docs/en/intro-to-loom.md [hunk 2]
```diff
@@ -185,3 +185,126 @@ yarn serve
...
+
+Wrapping it up, your `truffle-config.js` file should look something like this:
+
+```js
+
+import { sha256 } from 'js-sha256'
+import { CryptoUtils } from 'loom-js'
+import { LoomTruffleProvider } from 'loom-truffle-provider'
+
+const mnemonic = 'YOUR MNEMONIC'
```
Comment (enlight) [inline docs/en/intro-to-loom.md]: This doesn't look right, the mnemonic words need to be converted to a 64-byte seed, then you SHA256 the seed to get the 32 bytes to pass into `generatePrivateKeyFromSeed`

Comment (mimiiiz) [reply docs/en/intro-to-loom.md]: `sha256.array(mnemonic)`  will returns 32 bytes

Comment (andreipope) [reply docs/en/intro-to-loom.md]: @enlight @mimiiiz can't we just use:
```
loomTruffleProvider.createExtraAccountsFromMnemonic(mnemonic, 10);
```

Comment (enlight) [reply docs/en/intro-to-loom.md]: Yes, that `createExtraAccountsFromMnemonic` seems reasonable, @mimiiiz can you make the dashboard generate the key like LoomTruffleProvider? https://github.com/loomnetwork/loom-truffle-provider/blob/c5a56aa2e4fda2317be6ca84e736e0f8e28070ff/src/index.ts#L45

**Comment by andreipope** [comment]
@mimiiiz @enlight I did a couple of changes (removed the amount, and the way in which we compute the private key). If you'd like to use `createExtraAccountsFromMnemonic` instead, it'll take a few minutes to update the doc. Thanks

File: docs/en/intro-to-loom.md [hunk 3]
```diff
@@ -185,3 +185,129 @@ yarn serve
 2. Also, is possible to call truffle directly with `yarn truffle`.
 
 3. We're not versioning the build directory for this particular example, although is recommended to do so. The limitation can be removed by editing the `.gitignore` file.
+
+
+## Deploying to Loom
```
Comment (mattkanwisher) [inline docs/en/intro-to-loom.md]: Loom mainnet

Comment (mattkanwisher) [reply docs/en/intro-to-loom.md]: there are a ton of additional steps that are only needed for mainnet, maybe break to a second page?

Comment (andreipope) [reply docs/en/intro-to-loom.md]: @mattkanwisher moved it to a separate page and reordered the items. Thoughts?

Screenshot 2019-06-27 13 26 08

Comment (mattkanwisher) [reply docs/en/intro-to-loom.md]: put mainnet last

File: docs/en/deploy-loom-mainnet.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+Next, copy the public address and paste it into the "Your Loom Public Address" textbox:
+
+![Public Address](/developers/img/deploy-to-loom-paste-public-address.png)
+
+Lastly, click "Add Key" and you're set.
+
+#### Option 2 - Use an Existing Address
+
+Now, as an example, let's suppose you already have a Loom address- `0xC8613234a93977079d464e9b77bF60b227DB67A1`. If so, all you have to do is to replace the `0x` prefix with `loom`.  It'll look something like this: `loomC8613234a93977079d464e9b77bF60b227DB67A1`.
```
Comment (enlight) [inline docs/en/deploy-loom-mainnet.md]: I don't think this prefix replacing is needed anymore, the dashboard should accept both `0x` & `loom` prefixes already (for adding keys anyway).

File: docs/en/intro-to-loom.md (hunk 3 above)
Comment (mattkanwisher) [reply docs/en/intro-to-loom.md]: fixed

**Pull request merged**